src/main/resources/io/jenkins/plugins/pipelinegraphview/PipelineGraphViewConfiguration/config.jelly

```diff
@@ -3,11 +3,24 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
   <f:section title="${%Pipeline graph view}">
+    <link rel="stylesheet" href="${resURL}/plugin/pipeline-graph-view/css/settings.css" type="text/css" />
+
     <f:entry>
-      <f:checkbox field="showGraphOnJobPage" title="${%Show pipeline graph on job page}" />
+      <f:checkbox field="showGraphOnJobPage" title="${%Show pipeline graph on job page}" name="pgv.showGraphOnJobPage" />
     </f:entry>
+    <div class= "pgv-nested-checkbox-group" data-module="collapsable" data-controlled-by="pgv.showGraphOnJobPage">
```

Comment:
shouldn't use custom CSS for this, should probably be a `f:optionalBlock`: 
https://reports.jenkins.io/core-taglib/jelly-taglib-ref.html#form:optionalBlock  (anchor link isn't working, you'll have to search the pagE)

Reply:
ahh missed, that I thought there must be something in core but I couldn't find it